_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -87,15 +87,23 @@ newRow <- list("tortoiseshell", 3.3, TRUE, 9)
 cats <- rbind(cats, newRow)
 ```
 
+Looks like our attempt to `rbind` returns a warning.  Recall that, unlike errors, warnings do not necessarily stop a function from performing its intended action.  You can confirm this by taking a look at the `cats` data frame.
```

Comment:
Usually the style we follow suggests functions are specified with their `()`, so I might suggest changing "attempt to `rbind` returns" to "attempt to use the `rbind()` function returns".

Reply:
Hi @naupaka.  Thanks for the comment - I should have been paying more attention to the style guide.  I've added a commit that uses the language you suggested.